setup.py

```diff
@@ -0,0 +1,13 @@
+from distutils.core import setup
+setup(
+    name='armus',
+    packages=['armus'],
```

Comment:
Here's the documentation on what you need to do: https://docs.python.org/2/distutils/setupscript.html#installing-scripts